src/main/java/org/cactoos/list/Joined.java

```diff
@@ -53,19 +53,21 @@ public Joined(final List<X>... src) {
      * @param items List
      * @since 0.32
      */
+    @SuppressWarnings("unchecked")
     public Joined(final X item, final List<X> items) {
-        super(() -> new Joined<X>(new ListOf<X>(item), items));
+        super(new Joined<>(new ListOf<>(item), items));
     }
 
     /**
      * Ctor.
      * @param src Source lists
      */
     public Joined(final Iterable<List<X>> src) {
-        super(() -> Collections.unmodifiableList(
-            new ListOf<>(src).stream()
-                .flatMap(List::stream)
-                .collect(Collectors.toList())
+        super(
+            Collections.unmodifiableList(
```

Comment:
@victornoel Cactoos follows the *No code in constructors* principle. Thus, if you changed this piece of code, we'll haven't a "lazy initilization" object. My suggestion is come back the old code.

Reply:
@fabriciofx before the constructor of the super class was calling `value()` on the scalar, so in practice this was the same as what is happening right now. More generally, your proposal is counter to what is being achieved here, see #1242 for more details on the direction being taken with collections in cactoos

Reply:
@victornoel Looks like we could use the result of https://github.com/yegor256/cactoos/pull/1253 (Immutable decorators) here, WDYT?

Reply:
@paulodamaso actually I kept the same code as before on purpose because I believe that what should happens is what is outlined in #1254.